python/tests/spark/sql/codegen/test_fs.py

```diff
@@ -104,8 +104,8 @@ def test_validate_yaml_spec(tmp_path):
 
     assert spec.name == "test_yaml_model"
     assert spec.model_uri == "s3://bucket/to/model.pt"
-    assert spec.pre_processing is not None
-    assert spec.post_processing is not None
+    assert spec.pre_processing is None
```

Comment:
As mentioned above, i think we were deliberately making pre/post non none value so that the callers (i.e., 3rd party integration) can be done easily.

Reply:
Answered above.